pkg/install/v1alpha1/cstor_volume.go

```diff
@@ -536,11 +557,9 @@ metadata:
   name: cstor-volume-list-listtargetservice-default
 spec:
   meta: |
-    {{- /*
-    Create and save list of namespaces to $nss.
-    Iterate over each namespace and perform list task
-    */ -}}
-    {{- $nss := .Config.RunNamespace.value | default "" | splitList ", " -}}
+    {{- $isClone := .Volume.isCloneEnable | default "false" -}}
+    {{- $runNamespace := printf "%s, %s" .Config.RunNamespace.value .Volume.runNamespace -}}
+    {{- $nss := $runNamespace | default "" | splitList ", " -}}
```

Comment:
@princerachit Have you tried using action: list with comma separated namespaces?
I guess, repeatWith is not required. jsonPath will get modified accordingly.

Reply:
I have not checked it. I will send this as an improvement in another PR where I address this for all runtasks